cg/apps/demultiplex/sample_sheet/sample_sheet_validator.py

```diff
@@ -10,29 +10,15 @@
 from cg.apps.demultiplex.sample_sheet.read_sample_sheet import (
     get_flow_cell_samples_from_content,
     get_raw_samples_from_content,
-    get_sample_type_from_content,
     validate_samples_unique_per_lane,
 )
-from cg.apps.demultiplex.sample_sheet.sample_models import (
-    FlowCellSample,
-    FlowCellSampleBcl2Fastq,
-    FlowCellSampleBCLConvert,
-)
+from cg.apps.demultiplex.sample_sheet.sample_models import FlowCellSample, FlowCellSampleBCLConvert
```

Comment:
```suggestion
from cg.apps.demultiplex.sample_sheet.sample_models import FlowCellSample, FlowCellSampleBCLConvert
```
FlowCellSampleBCLConvert into FlowCellSample?

Reply:
another PR. There are actually 4 more PRs to come. Otherwise we would have a giant PR